### DISCUSSION TITLE
Validation for a specific profile

### DISCUSSION THREAD

**Discussion by wq9578** [ZUGFeRD/mustangproject#306] [Ideas]
It would be desirable to validate for a specific profile, for example

`--action validate --format zf --version 2 --profile E`

for the ZUGFeRD v2 profile EN16931.

Until then, I would suggest to display an error message and return "invalid" as the result if a profile is specified, since running

`java -jar Mustang-CLI-2.6.0.jar --action validate --format zf --version 2 --profile E --source BASIC_Einfach.pdf`

yields a positive result although only the BASIC profile is implemented (`urn:cen.eu:en16931:2017#compliant#urn:factur-x.eu:1p0:basic`), but the parameters ask for the much more advanced profile EN16931.

**Comment by jstaerk** [comment]
`--action validate` already automatically recognizes the profile (by it's guideline ID) and acts accordingly, apart from the matching schema and schematron file it also decides if additionally the CEN validations are to take place (Basic, EN16931 and XRechnung) and, only in case of XRechnung, the additional Kosit validations. If you have a basic invoice which you consider wrongly treated please kindly open an issue including the steps to reproduce, i.e. attaching the the invoice, and the expected and actual results.
